File: 4. Functions/2. Java/voting.java
import java.util.Scanner;
public class voting {

    static void voting_age(int age){
        if(age >= 18){
            System.out.print(" YOU CAN VOTE NOW ");
        }
        else{
            System.out.print(" YOU CANNOT VOTE NOW ");
        }
    }

    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the Age : ");
        int age = sc.nextInt();
        voting_age(age);
        sc.close();
    }
}

File: 4. Functions/1. Cpp/voting.cpp
#include <bits/stdc++.h>
using namespace std;
#define ll long long
#define PI 3.1416

void send_help() {
	ios_base::sync_with_stdio();
	cin.tie();
	cout.tie();
#ifndef ONLINE_JUDGE
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);
#endif
}

void voting_age(int age) {
	if (age >= 18) {
		cout << "You can vote now" << endl;
	}
	else {
		cout << "First desire then deserve" << endl;
	}
}

int main() {
	send_help();
	int age;
	cin >> age;
	voting_age(age);
	return 0;
}